packages/react-reconciler/src/ReactFiberThrow.js

```diff
@@ -102,6 +102,7 @@ function createClassErrorUpdate(
   if (typeof getDerivedStateFromError === 'function') {
     const error = errorInfo.value;
     update.payload = () => {
+      logError(fiber, errorInfo);
```

Comment:
What if I have _both_ gDSFE and cDC? We'll want to log once.

Reply:
Seems like it would be worth calling `logError` in this case, so long as we check that cDC doesn't also exist.